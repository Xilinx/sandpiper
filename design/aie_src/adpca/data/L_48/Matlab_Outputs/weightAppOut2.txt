-6.131500602e-01 -1.771692276e+00
-1.354919624e+01 9.494402409e-01
1.544948816e+00 -3.117197514e+00
