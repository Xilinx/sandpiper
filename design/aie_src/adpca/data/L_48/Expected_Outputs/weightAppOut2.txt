-6.130506396e-01 -1.771597624e+00 
-1.354870033e+01 9.493677616e-01 
1.545007944e+00 -3.117179155e+00 
